www/locales/en.json

```diff
@@ -317,6 +317,7 @@
     },
     "SYNC_CONTROLLER": {
       "INTERNET_ERR": "Sorry! You seem to be offline. Sync only works when you are connected to the internet.",
+      "CODE_ERR": "Sorry! Failed to get a new code for you, please try again later.",
```

Comment:
I think there's a rule that we shouldn't use exclamation in text for the user.

Reply:
Oh, wasn't aware of that rule. Will change it. Where can I read these rules?